src/styles/colors.js

```diff
@@ -23,6 +23,7 @@ export default {
     darkDefaultButton: '#184E3D',
     darkDefaultButtonHover: '#2C6755',
     darkDefaultButtonPressed: '#467164',
+    darkRightHandSideOverlay: '#1A3D32',
```

Comment:
```suggestion
    darkRightHandPanelOverlay: '#1A3D32',
```

same for the other theme color